srcdeps-core/src/main/java/org/srcdeps/core/util/SrcdepsCoreUtils.java

```diff
@@ -178,6 +183,26 @@ public static boolean isWindows() {
         return isWindows;
     }
 
+    /**
+     * Opens am {@link InputStream} out of the given {@code url} and returns the content as a UTF-8 string.
+     *
+     * @param url
+     *            the {@link URL} to read from
+     * @return the content read from the given {@code url}
+     * @throws IOException
+     */
+    public static String read(URL url) throws IOException {
+        StringBuilder result = new StringBuilder();
+        try (Reader in = new InputStreamReader(url.openStream(), StandardCharsets.UTF_8)) {
+            char[] buf = new char[1024];
```

Comment:
1024 sounds too small for a buffer nowadays. Or do you expect the result to be small?

Reply:
er... would 4 KB be enough?

Reply:
The higher, the better :) Is this code here related to dependency downloading? If so, setting to 10k should be enough to read most meta-data files in one shot (pom files, for instance).

Reply:
> Is this code here related to dependency downloading?

No rather loading of small local files. Or loading resources from the class loader. I should actually look how big those files are and adjust appropriately.

Reply:
In the coming PR, I made a method argument out of the buffer so that the caller can dimension the buffer properly. It is also more efficient in that way because the buffer gets allocated only once for all files that are loaded in a loop. The files are up to 9.6 KB, so I am passing a 10KB buffer.